tests/integration_tests.py

```diff
@@ -0,0 +1,174 @@
+import json
+import requests
+
+base_url = "http://localhost:8080/ga4gh/drs/v1"
+data_objects_path = "data_objects.json"
+headers = {
+    'accept': 'application/json',
+    'Content-Type': 'application/json'
+}
+payload_objects = json.dumps({
+    "name": "Homo_sapiens.GRCh38.dna.chromosome.19.fa.gz",
+    "size": 16700000,
+    "created_time": "2021-05-04T11:59:56.400Z",
+    "updated_time": "2021-05-04T11:59:56.400Z",
+    "version": "38",
+    "mime_type": "application/json",
+    "checksums": [
+        {
+            "checksum": "18c2f5517e4ddc02cd57f6c7554b8e88",
+            "type": "md5"
+        }
+    ],
+    "access_methods": [
+        {
+            "type": "ftp",
+            "access_url": {
+                "url": "ftp://ftp.ensembl.org/pub/release-96/fasta/homo_sapien"
+                "s/dna//Homo_sapiens.GRCh38.dna.chromosome.19.fa.gz",
+                "headers": [
+                    "None"
+                ]
+            },
+            "region": "us-east-1"
+        },
+        {
+            "type": "ftp",
+            "access_url": {
+                "url": "ftp://ftp.ensembl.org/pub/release-96/fasta/homo_sapien"
+                "s/dna//Homo_sapiens.GRCh38.dna.chromosome.19.fa.gz",
+                "headers": [
+                    "None"
+                ]
+            },
+            "region": "us-east-1"
+        },
+    ],
+    "contents": [
+        {
+            "name": "Homo_sapiens",
+            "id": "b001",
+            "drs_uri": [
+                "drs://drs.example.org/314159",
+                "drs://drs.example.org/213512"
+            ],
+            "contents": []
+        }
+    ],
+    "description": "description",
+    "aliases": [
+        "alias1"
+    ]
+})
+payload_service_info = json.dumps({
```

Comment:
Can the payload JSON perhaps be externalized so that it can be easily reused and keep this module cleaner? See here, e.g.: https://github.com/elixir-cloud-aai/trs-filer/blob/dev/tests/mock_data.py